### DISCUSSION TITLE
luarocks command returns nil

### DISCUSSION THREAD

**Discussion by httpanand** [keplerproject/luarocks#1373] [Q&A]
luarocks command returns nil value . ! Please help

**Comment by daurnimator** [comment]
Which command? How did you install luarocks?
Please paste the full input/output.

**Reply by httpanand** [reply]
```
 > luarocks install http.request
stdin:1: syntax error near 'install'
 ```
or is this the syntax ? ``` luarocks = install http.request ```

**Reply by daurnimator** [reply]
Luarocks should be run from your terminal (e.g. inside of `bash` on linux/mac; or inside of cmd.exe on windows). Not from a lua interpreter.

**Reply by httpanand** [reply]
Im not able to install luarocks :/
the exe file opens and then automatically closes. nothing is installed !

**Reply by httpanand** [reply]
@daurnimator  I tried in Zerobrane and 
![image](https://user-images.githubusercontent.com/87514488/143899317-bd0e9fd3-aba9-444a-bc16-2c0acf4fadba.png)

**Reply by daurnimator** [reply]
> @daurnimator I tried in Zerobrane and ![image](https://user-images.githubusercontent.com/87514488/143899317-bd0e9fd3-aba9-444a-bc16-2c0acf4fadba.png)

You're typing that into a lua interpreter. You need to type it into cmd.exe.

**Reply by httpanand** [reply]
not able to install luarocks.. 👋🏽 sad
